api/tests/unit/application/admin-members/index_test.js

```diff
@@ -40,6 +40,143 @@ describe('Unit | Application | Router | admin-members-router', function () {
     });
   });
 
+  describe('POST /api/admin/admin-members', function () {
+    context('when user has role "SUPER_ADMIN"', function () {
+      it('should return a response with an HTTP status code 201', async function () {
+        // given
+        const adminMember = domainBuilder.buildAdminMember();
```

Comment:
Si on supprime `pre: [{ method: securityPreHandlers.checkUserHasRoleSuperAdmin }], les tests unitaires passent toujours, j'ai le même souci sur ma PR. Faudrait qu'on essaye de revoir l'implémentation de ces tests

Reply:
Le seul moyen de vérifier si le test passe c'est de remplacer l'utilisation de `securityPreHandlers.checkUserHasRoleSuperAdmin` par `securityPreHandlers.checkUserHasRoleCertif` ou un autre.

On ne teste pas si on a un (ou plusieurs) security prehandlers. Si on le fait il faudra surement faire un stub de hapi (je pense).

Reply:
On peut faire :
`expect(securityPreHandlers.checkUserHasRoleSuperAdmin).to.have.be.called;`

et là, si on supprime le pre handler, le test est rouge

Reply:
Good point!